Makefile

```diff
@@ -165,6 +165,16 @@ check-scripts:
     # Fail if any of these files have warnings
 	scripts/shellcheck.sh
 
+## check-version:           run check to ensure version in CHANGELOG.md matches references in files
+.PHONY: check-version
+check-version:
+    # Fail if syncing version would produce changes
```

Comment:
nit: comment should be at same indent level as command

Reply:
Done, and there was one other similar comment that I corrected as well.